runtime/src/launch/slurm-srun/launch-slurm-srun.c

```diff
@@ -228,6 +230,14 @@ static char* chpl_launch_create_command(int argc, char* argv[],
     nodeAccessStr = "exclusive";
   }
 
+  if (memEnv == NULL) {
+    memStr = "0";
```

Comment:
If it's not too tedious, a comment here along the lines of "0 means all" would be nice.

Reply:
Added